src/App.jsx

```diff
@@ -13,6 +14,9 @@ const App = () => {
         <Routes>
           <Route path='/' element={<Home />} />
           <Route path='/login' element={<Login />} />
+          <Route path='/freeboard' element={<FreeBoard />} />
+          <Route path='/profile' element={<Home />} />
+          <Route path='/FeedDetail' element={<Home />} />
```

Comment:
```suggestion
          <Route path='/feeddetail/:id' element={<Home />} />
```
- 대문자로 작성된 path 소문자로 변경하였고 디테일은 추 후에 백엔드에서 넘겨준 id 또는 다른 값으로 해당 게시글에 대한 정보를 받아와야 하므로 파라미터를 추가해 주어야 합니다 👍

Reply:
소문자로 변경 완료, id 파라미터 추가 완료 했습니다🙂